tensorboard/program.py

```diff
@@ -104,6 +104,18 @@ def __init__(
             from tensorboard import default
 
             plugins = default.get_plugins()
+        if assets_zip_provider is None:
+            try:
+                from tensorboard import assets
+            except ImportError as e:
+                # `tensorboard.assets` is not a strict Bazel dep; clients are
+                # required to either depend on `//tensorboard:assets_lib` or
+                # pass a valid assets provider.
+                raise ImportError(
```

Comment:
No action required (or just relief):

I was a bit worried program.py would have hard dep on `webfiles.zip` but this of course make it better.

Reply:
Yeah—it took a bit of thinking to come up with a build/import structure
that I liked, but I think that it turned out okay.